### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by charapennikaurm**

### CODE REVIEW

File: UI/css/style.css [hunk 1]
```diff
@@ -0,0 +1,450 @@
+body {
```
Comment (smiakchilo) [inline UI/css/style.css:1]: I believe the UI part does not belong to the task 4

File: UI/js/parallax.js-1.5.0/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (smiakchilo) [inline UI/js/parallax.js-1.5.0/LICENSE:1]: Avoid adding 3rd party code to PR and probably to repository as well (if it's an NPM dependency, the mention in your project's `package.json` is enough)

**Review by smiakchilo — CHANGES_REQUESTED** [review]

**Comment by smiakchilo** [comment]
Please always add meaningful commit messages (like "What exactly was added for what reason", up to 80-100 characters

File: UI/css/style.css (hunk 1 above)
Comment (charapennikaurm) [reply UI/css/style.css:1]: I'm still waiting for response for pull request with UI and hadn't figured out how to select commits for pull request

**Comment by smiakchilo** [comment]
Easy)
1) You create the new branch not from the previous branch but from the master. I.e., first checkout master, then create new branch with git checkout -b “task4” etc. (or do the same via the IDEA interface)

2) If you  already committed to a “wrong” branch, you: 

- check out master

- create new branch

- do cherry-pick of only the needed commits from the wrong to the new branch

**Comment by charapennikaurm** [comment]
I have deleted UI files form this pull request, can I merge it now?

**Comment by smiakchilo** [comment]
Yep, go ahead)

**Comment by charapennikaurm** [comment]
Thank you)

**Pull request merged**